contents/handbook/growth/marketing/posthog-style-guide.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Style guide
```

Comment:
Is there a reason we can't just point to this existing style guide, which is already automatically added on every PR in this repo? 

https://github.com/PostHog/posthog.com/blob/master/STYLEGUIDE.md

Reply:
I'd suggest we cetralize things. We really want to have a consistent tone of voice and style between both the docs and the blog, so having two style guides doesn't make sense and means there's another thing to maintain, monitor, reference. 

I don't have a strong feeling on which it should be -- we could change the existing style guide to just be a link to this page -- but I do feel somewhat strongly that we should have one guide, not two.

Reply:
Fair. This is just an expansion of that guide, so we can just link to this one instead.